builds/build-strategies.adoc

```diff
@@ -7,6 +7,7 @@ toc::[]
 The following sections define the primary supported build strategies, and how to
 use them.
 
+////
 include::modules/builds-strategy-docker-build.adoc[leveloffset=+1]
```

Comment:
I commented out the docker build strategy for now and am going to follow up on if it can be removed. It's still in the code and in several places throughout the builds book.